worker/migrations/migrate-accounts-to-organisation.go

```diff
@@ -76,12 +87,45 @@ func buildCipher(orgKey []byte, slug string, acc *account.Account, url string, l
     md := metadata.New()
     md.DocTypeVersion = bitwarden.DocTypeVersion
 
+    bitwardenFields := make([]bitwarden.Field, 0)
+
+    for name, rawValue := range auth {
+        value, ok := rawValue.(string)
+        if (!ok) {
+            continue
+        }
+        if !isAdditionalField(name) {
+            continue
+        }
+
+        ivName := crypto.GenerateRandomBytes(16)
+        encName, err := crypto.EncryptWithAES256HMAC(key, hmac, []byte(name), ivName)
+        if err != nil {
+            return nil, err
+        }
+
+        ivValue := crypto.GenerateRandomBytes(16)
+        encValue, err := crypto.EncryptWithAES256HMAC(key, hmac, []byte(value), ivValue)
+        if err != nil {
+            return nil, err
+        }
+
+        logger.Infof("Adding field %s = %s", name, value)
```

Comment:
I am not comfortable with this log: won't it display potential private data in plaintext?

Reply:
It has been removed in later commits.